### PR TITLE
[IA-4039] Make cbas/ui/wds/cromwell disable-able on the GCP side

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-4039

Separate from https://github.com/broadinstitute/cromwhelm/pull/192 to not muddy the waters.

We previously made things disablable for Azure -- doing similar for GCP.

### CODE REVIEW

File: terra-batch-libchart/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ exports:
       name: coa-cromwell-svc
       image: broadinstitute/cromwell:85-46a7918
       port: 8000
-      coaEnabled: true
+      enabled: true
```
Comment (rtitle) [inline terra-batch-libchart/values.yaml:7]: Changed `coaEnabled` to just `enabled` in the lib chart, and make it control both sub-charts. Default all to `true` -- Leo will enable/disable a different set of services depending on environment/cloud.

**Comment by rtitle** [comment]
Tested this on a BEE (GCP and Azure) and seems to work. Requesting thumbs

**Review by LizBaldo — APPROVED** [review]

File: coa-helm/templates/cbas-api.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.cbas.coaEnabled }}
+{{- if .Values.cbas.enabled }}
```
Comment (aaronkanzer) [inline coa-helm/templates/cbas-api.yaml:1]: Not sure if a corresponding Leo PR is coming up, but might want to change these values? https://github.com/DataBiosphere/leonardo/pull/3128/files#r1103089975

**Review by jgainerdewar — APPROVED** [review]

**Comment by rtitle** [comment]
Tested this on a BEE for GCP and Azure

File: coa-helm/templates/cbas-api.yaml (hunk 2 above)
Comment (rtitle) [reply coa-helm/templates/cbas-api.yaml:1]: Yup! I did that locally - going to update the generated PR when it comes

**Comment by rtitle** [comment]
Hopefully this will work because there's a JIRA ticket!

**Pull request merged**